code/game/objects/structures/beds_chairs/chair.dm

```diff
@@ -385,17 +391,21 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/structure/chair/stool/bar, 0)
 /obj/item/chair/hit_reaction(mob/living/carbon/human/owner, atom/movable/hitby, attack_text = "the attack", final_block_chance = 0, damage = 0, attack_type = MELEE_ATTACK, damage_type = BRUTE)
 	if(attack_type == UNARMED_ATTACK && prob(hit_reaction_chance) || attack_type == LEAP_ATTACK && prob(hit_reaction_chance))
 		owner.visible_message(span_danger("[owner] fends off [attack_text] with [src]!"))
+		if(prob(break_chance + damage + 10)) //We want to at least ensure that there is always a minimum chance of it breaking to prevent infinite blocks
+			smash(owner)
 		return TRUE
 	return FALSE
 
 /obj/item/chair/afterattack(atom/target, mob/user, click_parameters)
 	if(!prob(break_chance))
 		return
 	user.visible_message(span_danger("[user] smashes [src] to pieces against [target]"))
-	if(iscarbon(target))
-		var/mob/living/carbon/C = target
-		if(C.health < C.maxHealth*0.5)
-			C.Paralyze(20)
+	if(ishuman(target) && !HAS_TRAIT(target, TRAIT_BRAWLING_KNOCKDOWN_BLOCKED))
+		var/mob/living/carbon/human/give_this_fucker_the_chair = target
+		if(check_behind(user, give_this_fucker_the_chair) || give_this_fucker_the_chair.get_timed_status_effect_duration(/datum/status_effect/staggered))
+			give_this_fucker_the_chair.Knockdown(2 SECONDS)
+			if(inflicts_stun_vulnerability)
+				give_this_fucker_the_chair.apply_status_effect(/datum/status_effect/next_shove_stuns)
```

Comment:
if they're under half health make them concussed (confused) because funny

Reply:
did this